internals/cli/masker/writer_test.go

```diff
@@ -271,6 +271,35 @@ func TestNewMaskedWriter(t *testing.T) {
 	}
 }
 
+func TestNewMaskedWriter_OnlyFlushingOnTimeoutBug(t *testing.T) {
```

Comment:
Every test asserts that there is not a bug. Therefore, I think we should explain the test instead of the bug in the function name.

Reply:
e.g. `FlushBeforeTimeout`

Reply:
:heavy_check_mark: